addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/handler/YeelightHandlerBase.java

```diff
@@ -123,6 +123,34 @@ public void channelLinked(ChannelUID channelUID) {
         }
     }
 
+    public void handleCommandHandler(ChannelUID channelUID, Command command, String logInfo) {
+        logger.debug(logInfo, command);
+
+        // If device is disconnected, start discover to reconnect.
+        if (mDevice.isAutoConnect() && mDevice.getConnectionState() != ConnectState.CONNECTED) {
+            DeviceManager.getInstance().startDiscovery(5 * 1000);
+            return;
```

Comment:
Is return the right thing to do here (commands will get ignored then)? Also is this part needed anymore when there is a handler for REFRESH commands?

Reply:
No, return is imho not correct. But I have really no clue how the whole communication works; all those calls to `startDiscovery` look pretty fishy to me...